builders/build/customize/.pages

```diff
@@ -1,8 +1,9 @@
-title: Local Environment
+title: Customize Chain
 hide: false
 nav:
   - index.md
   - 'Prerequisites': 'prerequisites.md'
   - 'Adding a Built-in Module': 'adding-built-in-pallet.md'
```

Comment:
This "pallet" filename slipped away :S, we should change it to "module" to have the url consistent with the title & general content

Reply:
Agree! Fixed